client/src/index.css

```diff
@@ -6,3 +6,11 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+.site {
+  display: flex;
+  min-height: 75vh;
+  flex-direction: column;
+}
+.site-content {
+  flex: 1;
```

Comment:
I think the problem with this is that the navbar is now aligned to the left. It was better before when it was more centrally aligned. Can you try to remove this to see if it's still work? (navbar more centrally aligned on the screen and footer at the bottom)

Reply:
it is removed